tests/utils/scribereader_test.py

```diff
@@ -3,12 +3,83 @@
 
 import pytest
 
+import tron.utils.scribereader
 from tron.utils.scribereader import read_log_stream_for_action_run
 
 try:
     import scribereader  # noqa: F401
+    from clog.readers import S3LogsReader  # noqa: F401
 except ImportError:
-    pytest.skip("scribereader not available, skipping tests", allow_module_level=True)
+    pytest.skip("yelp logs readers not available, skipping tests", allow_module_level=True)
+
+
+def static_conf_patch(args={}):
+    return lambda arg, namespace, default=None: args.get(arg)
+
+
+def test_read_log_stream_for_action_run_not_available():
+    tron.utils.scribereader.scribereader_available = False
+    tron.utils.scribereader.s3reader_available = False
```

Comment:
i *think* setting these with a mock might be a tad cleaner since that would take care of setting things back without needing a finally block